src/main/java/Duke.java

```diff
@@ -1,10 +1,107 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    private static final int TODO_DONE_LENGTH = 5;
+    private static final int DEADLINE_LENGTH = 9;
+    private static final int EVENT_LENGTH = 6;
+    private static final int BY_AT_LENGTH = 3;
+    private static Task[] tasksList = new Task[100];
+    private static int taskCounter = 0;
+
     public static void main(String[] args) {
+        showWelcomeMessage();
+        inputLoop();
+    }
+
+    private static void inputLoop() {
+        Scanner in = new Scanner(System.in);
+        String line;
+        while (true) {
+            line = in.nextLine();
+            if (line.equals("bye")) {
+                showExitMessage();
+                break;
+            } else if (line.equals("list")) {
+                printTasksList();
+                continue;
+            } else if (line.startsWith("done")) {
+                int index = Integer.parseInt(line.substring(TODO_DONE_LENGTH)) - 1;
+                tasksList[index].markAsDone();
+                printDoneTask(index);
+                continue;
```

Comment:
continue appears here but not in subsequent else if statements, perhaps you could standardize across the whole while loop?

Reply:
thank you, yes i will try to standardize it